_data/whitelist.yml

```diff
@@ -109,7 +109,7 @@ projects:
 - name: SiDock@Home
   link: https://www.sidock.si/sidock/
   goal: Works on drug discovery with COVID-19 as its primary focus
-  sponsor: COVID.SI project
+  sponsor: COVID.SI project. Accepts Gridcoin donations
```

Comment:
is this the right place for this?

Reply:
The other projects on the whitelist mention that they accept BTC donations. If that is relevant, I think we should mention projects that accept GRC donations.

Reply:
yoyo also accepts Gridcoin. If we add it here it should be there as well

Reply:
Cool didn't know that, adding now

Reply:
added

Reply:
Sidock accepts BTC too. If yoyo says both, so should this one

Reply:
Added

Reply:
Added